opacus/accountants/accountant.py

```diff
@@ -15,5 +15,10 @@ def get_epsilon(self, delta: float, *args, **kwargs) -> float:
         pass
 
     @abc.abstractmethod
-    def __len__(self):
+    def __len__(self) -> int:
+        pass
+
+    @classmethod
```

Comment:
nit: also add `@property`?

Reply:
Would be good, but folks say this [only works only in py3.9+](https://stackoverflow.com/a/1800999)

Reply:
Ugh! OK. Oh well!